src/AspNet.Security.OAuth.SuperOffice/SuperOfficeAuthenticationOptions.cs

```diff
@@ -0,0 +1,177 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System;
+using System.Globalization;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using Microsoft.IdentityModel.Protocols;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using Microsoft.IdentityModel.Tokens;
+
+namespace AspNet.Security.OAuth.SuperOffice
+{
+    /// <summary>
+    /// Defines a set of options used by <see cref="SuperOfficeAuthenticationHandler"/>.
+    /// </summary>
+    public class SuperOfficeAuthenticationOptions : OAuthOptions
+    {
+        private SuperOfficeAuthenticationEnvironment _environment;
+
+        /// <summary>
+        /// Initializes a new <see cref="SuperOfficeAuthenticationOptions"/>. Default environment is Development (SOD).
+        /// </summary>
+        public SuperOfficeAuthenticationOptions()
+        {
+            Environment = SuperOfficeAuthenticationEnvironment.Development;
+            CallbackPath = SuperOfficeAuthenticationDefaults.CallbackPath;
+            Scope.Add("openid");
+
+            ClaimActions.MapJsonKey(SuperOfficeAuthenticationConstants.ClaimNames.AssociateId, SuperOfficeAuthenticationConstants.PrincipalNames.AssociateId);
+            ClaimActions.MapJsonKey(SuperOfficeAuthenticationConstants.ClaimNames.UserPrincipalName, SuperOfficeAuthenticationConstants.PrincipalNames.Associate);
+            ClaimActions.MapJsonKey(SuperOfficeAuthenticationConstants.ClaimNames.Email, SuperOfficeAuthenticationConstants.PrincipalNames.EMailAddress);
```

Comment:
Sorting:

```suggestion
            ClaimActions.MapJsonKey(SuperOfficeAuthenticationConstants.ClaimNames.AssociateId, SuperOfficeAuthenticationConstants.PrincipalNames.AssociateId);
            ClaimActions.MapJsonKey(SuperOfficeAuthenticationConstants.ClaimNames.Email, SuperOfficeAuthenticationConstants.PrincipalNames.EMailAddress);
            ClaimActions.MapJsonKey(SuperOfficeAuthenticationConstants.ClaimNames.UserPrincipalName, SuperOfficeAuthenticationConstants.PrincipalNames.Associate);
```

Reply:
I do not think you have the latest version any this project... Sorting of the Constants has already been done. 

I also see that you reference System.IdentityModel.Tokens.Jwt in the csproj file, which is no longer there...

Please fetch latest before commenting.

Reply:
I reviewed the entire diff in the GitHub UI without any filtering.

Reply:
This makes no sense.
![image](https://user-images.githubusercontent.com/1012665/81932176-a79a9380-95eb-11ea-8393-04878eb9c3ca.png)

Reply:
ClaimActions sorted.

Reply:
I think you confused two different comments here. Your screenshot above is the package references, but this comment thread is about the tests.

Anyway, I've fixed the sorting myself here.